src/main/java/com/julienvey/trello/domain/TrelloEntity.java

```diff
@@ -10,7 +10,16 @@
     @JsonIgnore
     protected Trello trelloService;
```

Comment:
We should consider to make this field private to force subclasses use `getTrelloService()` method.

Reply:
We can mark this field as deprecated.